### DISCUSSION TITLE
Как использовать переключение чата при помощи Chatty?

### DISCUSSION THREAD

**Discussion by BlxckFox** [TownyAdvanced/Towny#7368] [Q&A]
Ru
Добрый день. Подскажите пожалуйста как я могу вернуть привычный чат?
На сервере установлены Towny + Chatty 
Присутствует значение false в чат конфиге towny

Eng
Good afternoon. Please tell me how can I return the usual chat? The format is set to Towny + Chatty There is a false value in the Towny chat config

**Comment by LlmDl** [comment]
If you want to return to general chat use `/gc`
